database/migrations/001_init_database.go

```diff
@@ -0,0 +1,149 @@
+package migrations
+
+import (
+	"github.com/flashbots/mev-boost-relay/database/vars"
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+func GetInitDatabase() *migrate.Migration {
```

Comment:
```suggestion
func CreateInitialDatabase() *migrate.Migration {
```

Reply:
Why have a function here at all, instead of just `var Migration001InitDatabase = migrate.Migration{...`?